repxe_host.rb

```diff
@@ -252,12 +252,34 @@ def restart_host(entry)
 end
 
 def refresh_vault_keys
```

Comment:
There is already a refresh_vault_keys defined in `lib/cluster_data.rb`
https://github.com/bloomberg/chef-bach/blob/master/lib/cluster_data.rb#L120-L158